src/Tools/BuildUtil/BuildUtil/BuildUtil.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Roslyn.MSBuild.Util</RootNamespace>
     <AssemblyName>Roslyn.MSBuild.Util</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>net46</TargetFramework>
```

Comment:
Is there anything useful in this project that could be used by the .net core build? Or are we okay with this restriction (vs. splitting into another project). (I assume the ValidateBuildEnvironment.cs is incompatible with netstandard)